bundles/org.openhab.binding.emby/src/main/java/org/openhab/binding/emby/internal/EmbyBindingConstants.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.emby.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link EmbyBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Zachary Christiansen - Initial contribution
+ */
+@NonNullByDefault
+public class EmbyBindingConstants {
+
+    private static final String BINDING_ID = "emby";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_EMBY_CONTROLLER = new ThingTypeUID(BINDING_ID, "controller");
+    public static final ThingTypeUID THING_TYPE_EMBY_DEVICE = new ThingTypeUID(BINDING_ID, "device");
+
+    // List of all Channel ids
+    public static final String CHANNEL_MUTE = "mute";
+    public static final String CHANNEL_STOP = "stop";
+    public static final String CHANNEL_CONTROL = "control";
+
+    public static final String CHANNEL_TITLE = "title";
+    public static final String CHANNEL_SHOWTITLE = "show-title";
+
+    public static final String CHANNEL_MEDIATYPE = "media-type";
+
+    public static final String CHANNEL_CURRENTTIME = "current-time";
+
+    public static final String CHANNEL_DURATION = "duration";
+    public static final String CHANNEL_IMAGEURL = "image-url";
+    public static final String CHANNEL_IMAGEURL_TYPE = "image-url_type";
+    public static final String CHANNEL_IMAGEURL_MAXWIDTH = "image-url_maxWidth";
+    public static final String CHANNEL_IMAGEURL_MAXHEIGHT = "image-url_maxHeight";
+    public static final String CHANNEL_IMAGEURL_PERCENTPLAYED = "image-url_PercentPlayed";
```

Comment:
These seem like configuration parameters if i look at the thing-type.xml
1. parameters should have `camelCase` convention.
   For reference: https://www.openhab.org/docs/developer/guidelines.html#naming-convention
2. These parameters are not documented
3. How does this work ? i would not expect PercentPlayed as a configuration parameter. 
4. PercentPlayed is not listed in the thing-type.xml

Reply:
They are configuration parameters.   I will make the changes to add CONFIG in front of them and also add to the documentation

As to PERCENTPLAYED.  I will add to the readme to explain the function of this.  But it is a boolean parameter that allows the image url that is retrieved to have the following added onto the end 

Adding a Percent Played Indicator
A percent played overlay can be added onto the image using PercentPlayed=47

This changes the image that is retrieved.

Reply:
done